zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1333,6 +1334,7 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
                 rwc_inc: rwc_inc.clone(),
             },
         );
+        // TODO: constrain the value of rwc_inc.
```

Comment:
still an issue? @DreamWuGit

Reply:
This is not strictly required. That would be defence in depth in addition to the checks of the copy circuit.